### PR TITLE
docs(skills): add Fireworks Agent skill for customer fine-tuning

### PR DESCRIPTION
**Pull request by vivekchauhanFW**
## Summary

- Adds **`skills/fireworks-agent/SKILL.md`** — a customer-facing skill that drives the Fireworks Pilot Agent (`firectl session`) end-to-end from a single natural-language instruction.
- Designed to be loaded and followed by a customer's coding agent (Claude / Cursor / Codex / etc.) — includes a dedicated **"For coding agents"** section with a session-status state machine, cost-ceiling heuristics, polling cadence, stuck detection, verified text-parsing recipes, pre-flight checks, and account discovery.
- Documents the full lifecycle: `create` → `events --wait` → handle `waiting` → `update -n` → loop → `succeeded` / `failed` / `cancelled`, plus the confirmation rule on mutating commands (`update`, `cancel`, `delete`).
- Adds a pointer from the top-level README so customers can discover it next to the existing renderer / verifier / dev skills.

## Validation

Tested end-to-end on the `pyroworks` account, single SFT run, no shortcuts:

| Field | Value |
|---|---|
| Session ID | `1779157376-41e1` |
| Base model | `accounts/fireworks/models/qwen3-4b` |
| Dataset | `accounts/pyroworks/datasets/8b-sharegpt-subset-500-out` (500 rows) |
| Output model | `accounts/pyroworks/models/qwen3-4b-smoke-lr15e5-lora8-1779158908` |
| Job ID | `vfes4zbn` |
| Final train / val loss | 0.7044 / 0.7771 |
| Cost | $0.37 actual vs $0.42 estimated (-12%) |
| Wall clock | ~36 min (planning + ~5 min training) |

Every text-parsing recipe in the skill (`SESSION_ID`, `STATUS`, latest `status_info` block, `MODEL_ID`, `JOB_ID`, `ACCOUNT`) was verified against this real session.

## CLI gotchas surfaced (and documented in the skill)

Three real issues found while writing and validating — agents would have hit each one:

1. **`--dry-run` on `session create` actually creates and runs the session** despite the help text claiming it "prints the request proto without running it". Surfaced when our supposed dry-run returned a session ID that was already in `running` state and accruing compute.
2. **`session update -i ...` doesn't work** — fails with `unknown shorthand flag: 'i'`. The real flag is `-n / --instruction` (same as on `session create`). The internal Pilot Agent skill we inherited from had the same `-i` typo.
3. **`--output json` is silently ignored** on `session create` and `session list`, despite the help text advertising `text|json|flag` and `text|json` respectively. The output stays in text format. The skill explicitly tells agents not to plan on JSON parsing and ships working `awk` / `grep` recipes instead.

## Test plan

- [x] Validated end-to-end against a real 500-row chat dataset on qwen3-4b (above).
- [x] All text-parsing recipes verified against the real session.
- [x] CLI flag claims cross-checked against `firectl session <cmd> --help` for each subcommand.
- [x] Pre-flight account discovery recipe verified — returns `ACCOUNT=pyroworks` correctly.
- [x] Rendered through `grip` (GitHub's render API) for visual fidelity.
- [x] No lint errors.

Made with [Cursor](https://cursor.com)

---

> [!NOTE]
> **Low Risk**
> Low risk: documentation-only change adding a new skill guide and a README link; no runtime code paths or data handling are modified.
> 
> **Overview**
> Adds a new **`skills/fireworks-agent/SKILL.md`** operator guide for running end-to-end fine-tuning via the Fireworks Agent (`firectl session`), including session lifecycle steps (`create`/`events --wait`/`update`), cost-approval heuristics, text-parsing recipes, and failure/cancel/delete guidance.
> 
> Updates `README.md` to surface this new skill alongside the existing skills so customers/agents can discover it.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit f7d682aabf2ea880046ab2ff2694d82b2d825fed. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

**Review by pgarbacki — APPROVED** [review]

File: skills/fireworks-agent/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,627 @@
...
... the Fireworks Agent (a.k.a. Pilot Agent, exposed through `firectl session`). Use this skill whenever the user asks to fine-tune, SFT, DPO, ORPO, post-train, custom-train, auto-train, hyperparameter-tune, or hp-sweep a model on Fireworks — including `qwen3-4b`, `qwen3-8b`, `qwen3p5-9b`, `qwen3p5-27b`, `qwen3p5-35b-a3b`, `llama-v3p3-70b-instruct`, `kimi-k2p5`, `kimi-k2p6`, `glm-5p1`, `qwen3p5-397b-a17b`, or any other Fireworks-hosted base model. Also use it for kicking off a training session, monitoring or resuming a session, answering the agent's mid-run questions, approving training plans, troubleshooting `failed` sessions, listing/cancelling/deleting sessions, and figuring out how to call a fine-tuned model after training. Triggers on commands like `firectl session create / get / events / list / update / cancel / delete`, on terms like "Pilot Agent", "Fireworks Agent", "auto-tune on Fireworks", "fine-tune on Fireworks", and on session IDs of the form `1779157376-41e1`. Runs read commands (`create`, `get`, `events`, `list`) autonomously; routes mutating commands (`update`, `cancel`, `delete`) through user confirmation because they spend real money and influence training decisions.
```
Comment (yisu201506) [inline skills/fireworks-agent/SKILL.md:3]: Is fireworks agent only for training? I thought it is also for our fireworks platform/firectl.
